saltpack/saltpack.go

```diff
@@ -67,7 +67,7 @@ func VerifyDetachedCheckSender(message io.Reader, signature []byte, checkSender
 	kr := basic.NewKeyring()
 	var skey sp.SigningPublicKey
 	var err error
-	skey, _, err = sp.Dearmor62VerifyDetachedReader(message, string(signature), kr)
+	skey, _, err = sp.Dearmor62VerifyDetachedReader(sp.CheckKnownMajorVersion, message, string(signature), kr)
```

Comment:
Looks like we used to only accept V1 signatures, now we accept V1 and V2 signatures. But I think that's ok.